Source/Core/VideoCommon/TextureDecoder_Common.cpp

```diff
@@ -434,6 +434,11 @@ static inline u32 MakeRGBA(int r, int g, int b, int a)
   return (a << 24) | (b << 16) | (g << 8) | r;
 }
 
+static constexpr int DXTBlend(int v1, int v2)
```

Comment:
Just saw the c compare script, not sure how I missed it the first time!  Thanks @hthh !